src/ucsi/lpm/get_pdos.rs

```diff
@@ -171,9 +171,8 @@ impl Args {
         self
     }
 
-    pub fn source_capability_type(&self) -> SourceCapabilityType {
-        // Won't panic, validated in try_from
-        self.0.source_capability_type().try_into().unwrap()
+    pub fn source_capability_type(&self) -> Result<SourceCapabilityType, InvalidSourceCapabilityType> {
+        self.0.source_capability_type().try_into()
     }
```

Comment:
Same as `lpm::get_alternate_mode::Args`, this is technically unreachable but IMO it's better written as a plain ol struct despite that ultimately using 6 bytes instead of 4.